sdk/python/kfp/dsl/_metadata.py

```diff
@@ -48,10 +48,15 @@ def to_dict_or_str(self):
   @staticmethod
   def from_dict_or_str(json):
```

Comment:
Better to add some comments on what kinds of data can be passed to the method? It seems that the supported data is not just a json.

Reply:
Done.